.github/workflows/labeler.yml

```
@@ -0,0 +1,20 @@
+name: "System Probe PRs labeler"
```

I think we can simply name this:

```suggestion
name: "PR labeler"
```

since this may also be used by other teams in the future?